src/class.test.js

```diff
@@ -222,6 +222,39 @@ test('ctx.iopipe.metric adds metrics to the custom_metrics array', async () => {
     expect(_.isArray(metrics)).toBe(true);
     expect(metrics).toHaveLength(7);
     expect(metrics).toMatchSnapshot();
+
+    // Check for autolabel because metrics were added
+    const labels = _.chain(reports)
+      .find(obj => obj.aws.functionName === 'metric-test')
+      .get('labels')
+      .value();
+    expect(labels.includes('@iopipe/metrics')).toBe(true);
+  } catch (err) {
```

Comment:
Why catch if just rethrown?

Reply:
I admit to copy pasta-ing the style of the rest of the test. Because they are async functions, the convention it always to try/catch